jmh-benchmarks/src/main/java/org/apache/kafka/jmh/cache/LRUCacheBenchmark.java

```diff
@@ -35,14 +37,29 @@
  * http://hg.openjdk.java.net/code-tools/jmh/file/tip/jmh-samples/src/main/java/org/openjdk/jmh/samples/
  */
 @State(Scope.Thread)
+@OutputTimeUnit(TimeUnit.MILLISECONDS)
 public class LRUCacheBenchmark {
 
+    private static final int DISTINCT_KEYS = 10_000;
+
+    private static final String KEY = "the_key_to_use";
+
+    private static final String VALUE = "the quick brown fox jumped over the lazy dog the olympics are about to start";
+
+    private static final String[] KEYS = new String[DISTINCT_KEYS];
+
+    private static final String[] VALUES = new String[DISTINCT_KEYS];
+
     private LRUCache<String, String> lruCache;
 
-    private final String key = "the_key_to_use";
-    private final String value = "the quick brown fox jumped over the lazy dog the olympics are about to start";
     int counter;
 
+    static {
+        for (int i = 0; i < DISTINCT_KEYS; ++i) {
+            KEYS[i] = KEY + i;
+            VALUES[i] = VALUE + i;
+        }
+    }
 
     @Setup(Level.Trial)
     public void setUpCaches() {
```

Comment:
maybe rename this to `setUp` and put the loop populating the array in this method

Reply:
@bbejeck done :)